todo_app/app.py

```diff
@@ -2,19 +2,43 @@
 from todo_app.data.session_items import add_item, get_items
```

Comment:
When you are done with this module, you should be able to delete this line and delete the whole session_items.py file